### PR TITLE
MAT-3900 CQL Library Tab

### PR DESCRIPTION
**Pull request by praveenavas**
MAT-3900 CQL Library Tab

### CODE REVIEW

File: cypress/Shared/Header.ts [hunk 1]
```diff
@@ -3,5 +3,6 @@ export class Header {
     public static readonly signOutButton = '.sc-pVTFL'
     public static readonly measures = ':nth-child(2) > .sc-furwcr'
     public static readonly newMeasureButton = '[data-testid=create-new-measure-button]'
+    public static readonly cqlLibraryTab = ':nth-child(3) > .sc-furwcr'
```
Comment (cvasile001) [inline cypress/Shared/Header.ts:6]: can you request that the developer update this element with an ID?

File: cypress/integration/WebInterface/CQL Library/CQLLibrary.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import {OktaLogin} from "../../../Shared/OktaLogin"
```
Comment (cvasile001) [inline cypress/integration/WebInterface/CQL Library/CQLLibrary.ts:1]: This test is fine for now, but likely we will just update this for CQL name validations or something else as navigating to the tab should be inherent to most CQL Library test cases

**Review by cvasile001 — CHANGES_REQUESTED** [review]
see if you can get some IDs for the header elements

File: cypress/integration/WebInterface/CQL Library/CQLLibrary.ts (hunk 2 above)
Comment (praveenavas) [reply cypress/integration/WebInterface/CQL Library/CQLLibrary.ts:1]: Yeah this is a place holder for CQL Library tests. Will change the names going forward.

**Pull request closed without merging**